Client/src/Pages/Uptime/Monitors/index.jsx

```
@@ -123,12 +123,17 @@ const UptimeMonitors = () => {
 			</GenericFallback>
 		);
 	}
-	if (!isLoading && !monitorsAreLoading && totalMonitors === 0) {
+	if (true) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)